.github/workflows/build_and_release.yml

```diff
@@ -137,7 +139,7 @@ jobs:
           case ${ID} in
             'fedora')
           PACKAGE_TYPE='RPM';;
-            'ubuntu')
+            'ubuntu|debian')
```

Comment:
I think this needs to be specified like `'ubuntu'|'debian')`. That works in the build script in the containers at least.

Reply:
Hmmm, based upon the build failure, it looks like cpack doesn't come with TAR out of the box anymore: `CPack Error: Could not create CPack generator: TAR`. It might be worth changing the `PACKAGE_TYPE='TAR';;` statement below to be `PACKAGE_TYPE='TGZ;;`

Reply:
> Hmmm, based upon the build failure, it looks like cpack doesn't come with TAR out of the box anymore: CPack Error: Could not create CPack generator: TAR. It might be worth changing the PACKAGE_TYPE='TAR';; statement below to be PACKAGE_TYPE='TGZ';;

lol not sure how i didnt spot this before but cpack never came with TAR package type:
```
7Z - 7zip - (.7z)
TBZ2 (.tar.bz2)
TGZ (.tar.gz)
TXZ (.tar.xz)
TZ (.tar.Z)
TZST (.tar.zst)
ZIP (.zip)
```

Reply:
TAR issue is resolved on master. Within this PR i've changed the case statement and updated the readme ;)